var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -129,26 +129,28 @@ def install_targets(self):
         ]
         return make_args + self.make_defs
 
-    @run_after('install')
-    def check_install(self):
-        spec = self.spec
```

Comment:
please don't  comment out the whole thing. If the issue is with spec.cppflags` being undefined, then at least test it at the beginning of this function and return if that is the case.